src/elm/Page/Dashboard/Claim.elm

```diff
@@ -277,7 +277,12 @@ viewDetails shared model claim =
                         [ class "pt-2 text-body"
                         , classList [ ( "text-red line-through", isRejected ) ]
                         ]
-                        [ text (String.fromFloat claim.action.reward ++ " " ++ Eos.symbolToSymbolCodeString claim.action.objective.community.symbol) ]
+                        [ text
+                            (String.fromFloat claim.action.reward
+                                ++ " "
+                                ++ Eos.symbolToSymbolCodeString selectedCommunity
```

Comment:
yeah, here we might have a scenario of an user trying to open a link with a different community selected